File: app/front/src/components/main/dto/riyousha/getRiyoushaMasterResultDto.ts
import { FrameworkMessageAndResultDto, type FrameworkMessageAndResultDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { RiyoushaPartnerApiMasterEntity, type RiyoushaPartnerApiMasterEntityInterface } from "../../entity/riyoushaPartnerApiMasterEntity";
import { RiyoushaManagerMasterEntity, type RiyoushaManagerMasterEntityInterface } from "../../entity/riyoushaManagerMasterEntity";

interface GetRiyoushaMasterResultDtoInterface extends FrameworkMessageAndResultDtoInterface {
    /** 運営者マスタEntity */
    managerMasterEntity: RiyoushaManagerMasterEntityInterface;

    /** APIユーザマスタEntity */
    partnerApiMasterEntity: RiyoushaPartnerApiMasterEntityInterface;

}

class GetRiyoushaMasterResultDto extends FrameworkMessageAndResultDto
    implements GetRiyoushaMasterResultDtoInterface {

    /** 運営者マスタEntity */
    managerMasterEntity: RiyoushaManagerMasterEntityInterface;

    /** APIユーザマスタEntity */
    partnerApiMasterEntity: RiyoushaPartnerApiMasterEntityInterface;

    constructor() {
        super();

        /** 運営者マスタEntity */
        this.managerMasterEntity = new RiyoushaManagerMasterEntity();

        /** APIユーザマスタEntity */
        this.partnerApiMasterEntity = new RiyoushaPartnerApiMasterEntity();

    }
}

export { type GetRiyoushaMasterResultDtoInterface, GetRiyoushaMasterResultDto }

File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/riyousha/GetRiyoushaMasterResultDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.riyousha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkMessageAndResultDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.RiyoushaManagerMasterEntity;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.RiyoushaPartnerApiMasterEntity;

/**
 * 関連者マスタ取得結果Dto
 */
public class GetRiyoushaMasterResultDto extends FrameworkMessageAndResultDto // NOPMD DataClass
        implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 運営者マスタEntity */
    private RiyoushaManagerMasterEntity managerMasterEntity = new RiyoushaManagerMasterEntity();

    /** APIユーザマスタEntity */
    private RiyoushaPartnerApiMasterEntity partnerApiMasterEntity = new RiyoushaPartnerApiMasterEntity();

    /**
     * 運営者マスタEntityを取得する
     * 
     * @return 運営者マスタEntity
     */
    public RiyoushaManagerMasterEntity getManagerMasterEntity() {
        return managerMasterEntity;
    }

    /**
     * 運営者マスタEntityを設定する
     * 
     * @param managerMasterEntity 運営者マスタEntity
     */
    public void setManagerMasterEntity(final RiyoushaManagerMasterEntity managerMasterEntity) {
        this.managerMasterEntity = managerMasterEntity;
    }

    /**
     * APIユーザマスタEntityを取得する
     * 
     * @return APIユーザマスタEntity
     */
    public RiyoushaPartnerApiMasterEntity getPartnerApiMasterEntity() {
        return partnerApiMasterEntity;
    }

    /**
     * APIユーザマスタEntityを設定する
     * 
     * @param partnerApiMasterEntity APIユーザマスタEntity
     */
    public void setPartnerApiMasterEntity(final RiyoushaPartnerApiMasterEntity partnerApiMasterEntity) {
        this.partnerApiMasterEntity = partnerApiMasterEntity;
    }

}
